src/Tutor.Core/InstructorModel/InstructorService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using FluentResults;
+using Tutor.Core.DomainModel.KnowledgeComponents;
+using Tutor.Core.LearnerModel.DomainOverlay.EnrollmentModel;
+
+namespace Tutor.Core.InstructorModel;
+
+public class InstructorService : IInstructorService
+{
+    private readonly IInstructorRepository _instructorRepository;
+
+    public InstructorService(IInstructorRepository instructorRepository)
+    {
+        _instructorRepository = instructorRepository;
+    }
+    
+    public Result<List<Course>> GetCourses(int userId)
+    {
+        var instructor = _instructorRepository.GetByUserId(userId);
```

Comment:
As mentioned, maybe we can get the instructorId the same way we get the learnerId and store it in the JWT?